### PR TITLE
Sort /queues and /token

### PR DESCRIPTION
**Pull request by dhirenmathur**
Implements #76 

Commit contains changes to - 
Add explicit sorting logic to GET /queues and GET /tokens to return oldest first. 

* mvn package step: 
image

* Tested in postman: 
image
image

### CODE REVIEW

File: complete/src/main/java/com/example/restservice/service/QueueService.java [hunk 1]
```diff
@@ -78,9 +79,18 @@ public QueueStatusResponse getQueueStatus(String queueId) {
 
   @Transactional
   public MyQueuesResponse getMyQueues() {
-    return new MyQueuesResponse(
+    Stream<Queue> myQueueStream =
```
Comment (daltonfury42) [inline complete/src/main/java/com/example/restservice/service/QueueService.java:82]: can you do this without using this extra varaible?

File: complete/src/main/java/com/example/restservice/service/TokenService.java [hunk 2]
```diff
@@ -106,9 +108,18 @@ public TokenDetailResponse createToken(CreateTokenRequest createTokenRequest) {
...
             .findByOwnerId(loggedInUserInfo.getUserId())
+            .sorted(
+                new Comparator<Token>() {
+                  public int compare(Token a, Token b) {
+                    return a.getTokenCreationTimestamp().compareTo(b.getTokenCreationTimestamp());
+                  }
+                });
+
+    return new MyTokensResponse(
+        myTokenStream
```
Comment (daltonfury42) [inline complete/src/main/java/com/example/restservice/service/TokenService.java:122]: same as above, let's not use this extra variable

Comment (dhirenmathur) [reply complete/src/main/java/com/example/restservice/service/TokenService.java]: changed

File: complete/src/main/java/com/example/restservice/service/QueueService.java (hunk 1 above)
Comment (dhirenmathur) [reply complete/src/main/java/com/example/restservice/service/QueueService.java]: removed extra variable

File: complete/src/main/java/com/example/restservice/service/QueueService.java [hunk 3]
```diff
@@ -78,9 +79,16 @@ public QueueStatusResponse getQueueStatus(String queueId) {
 
   @Transactional
   public MyQueuesResponse getMyQueues() {
+
```
Comment (dhirenmathur) [inline complete/src/main/java/com/example/restservice/service/QueueService.java:82]:
```suggestion
```

File: complete/src/main/java/com/example/restservice/service/TokenService.java [hunk 4]
```diff
@@ -18,6 +18,8 @@
 import lombok.RequiredArgsConstructor;
 import org.apache.commons.lang3.ObjectUtils;
 import org.springframework.stereotype.Service;
+import java.util.Comparator;
+import java.util.stream.Stream;
```
Comment (dhirenmathur) [inline complete/src/main/java/com/example/restservice/service/TokenService.java:22]:
```suggestion
```

File: complete/src/main/java/com/example/restservice/service/QueueService.java [hunk 5]
```diff
@@ -18,6 +18,7 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.dao.DataIntegrityViolationException;
 import org.springframework.stereotype.Service;
+import java.util.stream.Stream;
```
Comment (dhirenmathur) [inline complete/src/main/java/com/example/restservice/service/QueueService.java:21]:
```suggestion
```

**Review by daltonfury42 — APPROVED** [review]
Enjoy your tshirt!

**Comment by dhirenmathur** [comment]
Haha this was just the first one. I've wanted to get started for a while, hacktoberfest is just an excuse :)

**Pull request merged**